.travis.yml

```diff
@@ -1,6 +1,8 @@
 # Copyright 2018 Stefan Seefeld
 # Distributed under the Boost Software License, Version 1.0.
 # (See accompanying file LICENSE_1_0.txt or copy at http://boost.org/LICENSE_1_0.txt)
+
```

Comment:
This file shouldn't contain any changes, other than additional build variants for C++17 - specific builds.